### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by kw9212**

### CODE REVIEW

File: contribution/plan.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Project Contributions
```
Comment (timkpaine) [inline contribution/plan.md:1]: remove this file or put contents in `ks3651.md`

File: contribution/plan.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+[First Contribution](https://github.com/danthareja/contribute-to-open-source/issues/1)
```
Comment (timkpaine) [inline contribution/plan.md:3]: pick at least one project and setup a markdown file for it, see other PRs as example

File: projects/javascript/To-do List.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+- Creating a To-do List
```
Comment (timkpaine) [inline projects/javascript/To-do List.md:1]: only need one idea, so pick your "best" for now

File: projects/python/project_ideas.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+Python
```
Comment (timkpaine) [inline projects/python/project_ideas.md:1]: only need one idea, so pick your "best" for now

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/javascript/To-do List.md (hunk 3 above)
Comment (kw9212) [reply projects/javascript/To-do List.md]: I picked one of the javascript projects for the contribution project and attached the link.

File: projects/python/project_ideas.md (hunk 4 above)
Comment (kw9212) [reply projects/python/project_ideas.md]: I picked one of the python projects for my project and attached the link.

File: contribution/plan.md (hunk 1 above)
Comment (kw9212) [reply contribution/plan.md]: I removed the "contributions" folder and left only the "projects" folder which has a javascript folder for the contribution and a python folder for the project.

I attached each link to the ks3651.md as the professor's md file.

File: contribution/plan.md (hunk 2 above)
Comment (kw9212) [reply contribution/plan.md]: I picked one of the javascript projects for the contribution project and attached the link as the professor's md file.

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**